src/styles/styles.scss

```diff
@@ -42,6 +42,7 @@ $BodyPaddingLeftRightMobile: calc((100% - #{$ContentBodyWidthMobile}) / 2);
 /**
  * this is to sync with when the navbar controls start to break
  */
+$PageBreakWidthWide: 1080px;
```

Comment:
Why here?  This looks like a home-page-specific thing.

Reply:
removed

Reply:
I understood the intention of Doug's comment to only move it to another place.
We probably need this variable in other places too